File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/scoreitmdto/i/IScoreItmDtoService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using iih.ci.mrqc.scoreitmdto.d;

namespace iih.ci.mrqc_stub.scoreitmdto.i
{
    public interface IScoreItmDtoService
    {
         ScoreItmDto[] getScoreItmDtos( String id_ent,String id_qc_type);
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/scoreitmdto/i/IScoreItmDtoService.java
package iih.ci.mrqc.scoreitmdto.i;

import iih.ci.mrqc.scoreitmdto.d.ScoreItmDto;
import xap.mw.core.data.BizException;


public interface IScoreItmDtoService {
	/**
	*  获取质控评分扣分项目DTO
	*/
    public abstract  ScoreItmDto[] getScoreItmDtos( String id_ent,String id_qc_type) throws BizException;
}
